awx/main/conf.py

```diff
@@ -775,6 +775,17 @@
     help_text=_('Indicates whether the instance is part of a kubernetes-based deployment.'),
 )
 
+register(
+    'SUBSCRIPTION_USAGE_MODEL',
+    field_class=fields.ChoiceField,
+    choices=[SUBSCRIPTION_USAGE_MODEL_UNIQUE_HOSTS, SUBSCRIPTION_USAGE_MODEL_UNIQUE_HOSTS_MONTHLY],
+    default='',
```

Comment:
default isn't in the choices, and that can cause problems in some corner cases. Some clients like to PATCH with defaults instead of resetting, which can give problems with stuff like this. So I think you just want to reuse one of those constants you have for the default.

Reply:
@AlanCoding added

Reply:
Anyway I suppose labels shouldn't contain words like "Automation Controller", "AoC on Azure" and other downstream terminology

Reply:
(SUBSCRIPTION_USAGE_MODEL_UNIQUE_HOSTS, _('Usage based on unique managed nodes in a large historical time frame and delete functionality for no longer used managed nodes')),
(SUBSCRIPTION_USAGE_MODEL_UNIQUE_HOSTS_MONTHLY, _('Usage based on unique managed nodes in a calendar month time frame')),


Somethin like this should work, you're right we shouldn't use product names here.

Reply:
Just as an FYI, Azure doesn't use calendar month, but uses a rolling month period based on installation date..

Reply:
@bhavenst oh is it? I was checking with Chris Hambridge that it was like this before, but then you moved to calendar month billing?

Reply:
Azure doesn't support calendar month. On AWS it is calendar month though AFAIK.

Reply:
@slemrmartin ok, lets take the SUBSCRIPTION_USAGE_MODEL_UNIQUE_HOSTS_MONTHLY out for now. @bhavenst AoC can set '' value for now.

Reply:
ok, done